### DISCUSSION TITLE
Add "Invert" Shortcuts to Multiselect

### DISCUSSION THREAD

**Discussion by KyleTryon** [charmbracelet/huh#564] [Ideas]
**Is your feature request related to a problem? Please describe.**
When using a multiselect component with a long list of options where you want to select many or all of the options, can be cumbersome as you must individually select each item.

**Describe the solution you'd like**
It can be helpful to be able to select all items in the list to quickly select and navigate from a long list. This also allows you to more easily deselect fewer options in the case you want to select most items but not all.

Add "Ctrl+a" for a `Select All` shortcut

**Describe alternatives you've considered**
Creating a new separate component with additional options

**Additional context**
An `Inverse Selection` addition may be good too, but could be addressed later.

**Source:** https://github.com/charmbracelet/huh/blob/main/field_multiselect.go
The [Keymap](https://github.com/charmbracelet/huh/blob/2ae64435bd5db02d08d3b01f911a0312d3517b57/keymap.go#L54)

**Comment by caarlos0** [comment]
select all/none has landed on main #325 

the inverse selection I'm not sure about... could you give some examples on how it would be useful?

thank you

**Comment by KyleTryon** [comment]
Awesome! 🎉 

For inverse, it can be helpful when selecting many items from a long list.

For example, say you are selecting multiple directories in a list:

```
- /dir/1
- /dir/2
- /dir/3
- /dir/4
- /dir/5
- /dir/6
- /dir/7
- /dir/8
```

You want to select all, besides 1,2,3.

One way to achieve this would be to select 1,2,3 and then invert the selection.
The other way (which btw I think is acceptable with select all being available), is to select all, then un-select 1,2,3.

**Comment by bashbunni** [comment]
Hey @KyleTryon I see what you mean. On `main` you can Select All or Deselect All, but not make a selection, then invert the selection.

I'm going to convert this to a discussion while we think on it. We will convert it back to an issue if we are ready to start working on it. Thanks for sharing new ways to improve Huh :)
